.devcontainer/devcontainer.json

```diff
@@ -5,6 +5,7 @@
 	},
 	"postCreateCommand": "npm install -g @devcontainers/cli",
 	"customizations": {
+		"editor.trimAutoWhitespace": true,
```

Comment:
customization to handle `whitespaces`

Reply:
Would that have to go under "vscode" > "settings"?

Reply:
Updated. Thank you also for the review!